workload-identity/bind-service-accounts.sh

```diff
@@ -19,7 +19,7 @@ set -o nounset
 set -o pipefail
 
 if [[ $# != 6 ]]; then
-  echo "Usage: $(basename "$0") <project> <zone> <cluster> <namespace> <name> <gcp-service-account>" >&2
+  echo "Usage: $(basename "$0") <project> <zone_or_region> <cluster> <namespace> <name> <gcp-service-account>" >&2
```

Comment:
The doc changes are good. I don't think there's need to churn the varaible name

Reply:
so `s/location/zone/g` and you're good?

(the line this comment is on will not change, which is my only confusion)

Reply:
sorry, yes <zone_or_region> in the comments sgtm!